Libs/MRML/Core/vtkDataFileFormatHelper.cxx

```diff
@@ -55,6 +55,7 @@ ITKImageFileFormat FileFormatTable[] =
   {"MGHImageIO", "Uncompressed FreeSurfer pixel data in binary", "MGH", ".mgh"},
   {"MGHImageIO", "Gzip Compressed FreeSurfer pixel data in binary", "MGH", ".mgz"},
   {"MGHImageIO", "Gzip Compressed FreeSurfer pixel data in binary", "MGH", ".mgh.gz"},
+  {"MINCImageIO", "Optional compression. Binary pixel data", "MINC", ".mnc"},
```

Comment:
compression have to be enabled though, through SetUseCompression / UseCompressionOn - methods of ImageIOBase